.lighthouserc.js

```diff
@@ -0,0 +1,234 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS-IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Config for lighthouse-ci
+ */
+
+ // To run Lighthouse Checks type lhci autorun
+ module.exports = {
+    ci: {
+      collect: {
+        numberOfRuns: 1,
+        startServerCommand: 'python -m scripts.start',
+        url: [
+          'http://127.0.0.1:8181/splash',
+          'http://127.0.0.1:8181/get_started',
+          'http://127.0.0.1:8181/teach',
+          'http://127.0.0.1:8181/library',
+          'http://127.0.0.1:8181/donate',
+          'http://127.0.0.1:8181/privacy',
+          'http://127.0.0.1:8181/contact',
+          'http://127.0.0.1:8181/about',
+          'http://127.0.0.1:8181/terms',
+          'http://127.0.0.1:8181/thanks',
+  
+        ],
+        settings: {
+          chromeFlags: ['--proxy-server=http://127.0.0.1:9999', '--allow-insecure-localhost'],
+        }
+      },
+      assert: {
+        assertMatrix: [
+          {
+            // All Webpages
+            matchingUrlPattern: '.*',
+            assertions: {
+              // Performance Audits
+              // maxNumericValue unit is in miliseconds
+              'first-contentful-paint': ['warn', {maxNumericValue: 12300}],
+              'first-meaningful-paint': ['warn', {maxNumericValue: 12800}],
+              'first-cpu-idle': ['warn', {maxNumericValue: 14600}],
+              'speed-index': ['warn', {maxNumericValue: 12300}],
+              'interactive': ['warn', {maxNumericValue: 15400}],
+              'max-potential-fid': ["warn", {maxNumericValue: 1300}],
+              // Performance Opportunities
+              // 'render-blocking-resources': ['warn', {minScore: 1}], // failing
+              'uses-responsive-images': ['warn', {minScore: 1}],
+              'offscreen-images': ['warn', {minScore: 1}],
+              // 'unminified-css': ['warn', {minScore: 1}], // failing 
+              // 'unminified-javascript': ['warn', {minScore: 1}], // failing
+              // 'unused-css-rules': ['warn', {minScore: 1}], // failing
```

Comment:
Remove these commented rules from this file.

Reply:
Done